rules-reviewed/quarkus/quarkus1-10/tests/quarkus1-11-resteasy-jackson-compiled.windup.test.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0"?>
+<ruletest id="quarkus1-11-resteasy-jackson-compiled-tests"
+          xmlns="http://windup.jboss.org/schema/jboss-ruleset"
+          xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+          xsi:schemaLocation="http://windup.jboss.org/schema/jboss-ruleset http://windup.jboss.org/schema/jboss-ruleset/windup-jboss-ruleset.xsd">
+    <testDataPath>data/test.jar</testDataPath>
+    <sourceMode>false</sourceMode>
+    <rulePath>../quarkus1-11-resteasy-jackson.windup.groovy</rulePath>
+    <ruleset>
+        <rules>
+            <rule id="quarkus1-11-resteasy-jackson-compiled-test">
```

Comment:
by convention we usually name tests the same as the rule they're testing with "-test" on the end